main.go

```diff
@@ -563,3 +568,56 @@ func deployCloudComponents(plan types.Plan) error {
 
 	return nil
 }
+
+func featureEnabled(features []string, secretFeatures []string) bool {
```

Comment:
How does this work?

Reply:
This is getting the list of features from the `Features []string` field which is part of `types.Plan` struct, but is not generated in the yaml file,  but instead is populated dynamically here `plan = filterFeatures(plan)`. So getting the list of enabled features and getting the list of filters which the secret has. Go through the features and check if they match directly return true, if not then feature  was not appended due to not configuring it/disabling it, so return false.

Reply:
I added unit tests to showcase how this works 👍